### PR TITLE
add user table

### LINKED ISSUE #8: WOS-D-001 系統需儲存使用者的相關資訊：登入 ID、姓名、密碼、電子郵件、地址

### PR DESCRIPTION
**Pull request by glu8637**

### CODE REVIEW

File: User_table.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE User(
```
Comment (nonorze-VOXX) [inline User_table.txt]: it is no work

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
user is not allow

File: User_table.txt (hunk 1 above)
Comment (glu8637) [reply User_table.txt]: try now

File: User_table.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE UserData(
+  ID int unique not null,
+  Name char[32] not null,
+  Password char[32] not null,
+  Email char[64] unique not null,
+  Address char[64] not null,
+  PhoneNumber char[16] not null,
+  Birthday timestamp,
+  Authority int not null
```
Comment (nonorze-VOXX) [inline User_table.txt]: these three is not in srs
why add these

File: User_table.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE UserData(
+  ID int unique not null,
+  Name char[32] not null,
+  Password char[32] not null,
+  Email char[64] unique not null,
+  Address char[64] not null,
```
Comment (nonorze-VOXX) [inline User_table.txt]: sql language should Uppercase

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]

File: User_table.txt (hunk 2 above)
Comment (nonorze-VOXX) [reply User_table.txt]: update: it is for someone want collect people private information
and Authority is for user role manage

**Review by orangeOAO — APPROVED** [review]

**Review by nonorze-VOXX — CHANGES_REQUESTED** [review]
new request: put in db folder and sub filename request .sql

File: User_table.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE UserData(
+  ID int unique not null,
```
Comment (wj461) [inline User_table.txt]: I think you can use serial replace int

File: User_table.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  Name char[32] not null,
```
Comment (wj461) [inline User_table.txt]: I think char[32] can use VARCHAR(32)

**Review by orangeOAO — APPROVED** [review]

**Review by orangeOAO — APPROVED** [review]

**Review by nonorze-VOXX — APPROVED** [review]
LGTM

**Pull request merged**